source/api.rst

```diff
@@ -26,6 +26,25 @@ You can refer to the next documentation to get the list of available endpoints.
 
 .. _websockets:
 
+Errors
+======
+
+Symbol uses conventional HTTP response codes to indicate the success or failure of an API request.
+
+Codes in the ``2xx`` range indicate success.
+Codes in the ``4xx`` range indicate an error occurred with the information provided by the user.
+Codes in the ``5xx`` range indicate an error with Symbol's server.
+
+.. csv-table:: HTTP Status Code Summary
+    :header: "Error Code", "Description"
```

Comment:
Add three columns "Error code"; "Response", "Description"

E.g.:   404; ResourceNotFound; The requested resource does not exist.